src/Desktop/Desktop.js

```diff
@@ -0,0 +1,130 @@
+  padding: 12px;
+  background: ${({ theme }) => theme.material};
+  border-top: 4px solid ${({ theme }) => theme.borderLightest};
+  border-left: 4px solid ${({ theme }) => theme.borderLightest};
+  border-bottom: 4px solid ${({ theme }) => theme.borderDark};
+  border-right: 4px solid ${({ theme }) => theme.borderDark};
+
+  outline: 1px dotted ${({ theme }) => theme.material};
+  outline-offset: -3px;
+  &:before {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    outline: 1px dotted ${({ theme }) => theme.material};
+  }
+  box-shadow: 1px 1px 0 1px ${({ theme }) => theme.borderDarkest};
+
+  &:after {
+    content: '';
+    display: inline-block;
+    position: absolute;
+    bottom: 4px;
+    right: 12px;
+    width: 10px;
+    border-top: 2px solid #4d9046;
+    border-bottom: 2px solid #07ff00;
+  }
+`;
+
+const Background = styled(StyledCutout).attrs(() => ({
+  'data-testid': 'background'
+}))`
+  width: 100%;
+  height: 100%;
+`;
+
+const Stand = styled.div`
+  box-sizing: border-box;
+  position: absolute;
+  top: calc(100% + 2px);
+  left: 50%;
+  transform: translateX(-50%);
+  height: 10px;
+  width: 50%;
+  background: ${({ theme }) => theme.material};
+  border-left: 2px solid ${({ theme }) => theme.borderLightest};
+  border-bottom: 2px solid ${({ theme }) => theme.borderDarkest};
+  border-right: 2px solid ${({ theme }) => theme.borderDarkest};
+  box-shadow: inset 0px 0px 0px 2px ${({ theme }) => theme.borderDark};
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: calc(100% + 2px);
+    left: 50%;
+    transform: translateX(-50%);
+    width: 80%;
+    height: 8px;
+    background: ${({ theme }) => theme.material};
+    border-left: 2px solid ${({ theme }) => theme.borderLightest};
+    border-right: 2px solid ${({ theme }) => theme.borderDarkest};
+    box-shadow: inset 0px 0px 0px 2px ${({ theme }) => theme.borderDark};
+  }
+  &:after {
+    content: '';
+    position: absolute;
+    top: calc(100% + 8px);
+    left: 50%;
+    transform: translateX(-50%);
+    width: 150%;
+    height: 4px;
+    background: ${({ theme }) => theme.material};
+    border: 2px solid ${({ theme }) => theme.borderDark};
+    border-bottom: none;
+    box-shadow: inset 1px 1px 0px 1px ${({ theme }) => theme.borderLightest},
+      1px 1px 0 1px ${({ theme }) => theme.borderDarkest};
+  }
+`;
+
+const Desktop = React.forwardRef(function Desktop(props, ref) {
+  const { backgroundStyles, children, ...otherProps } = props;
+
+  return (
+    <Wrapper ref={ref} {...otherProps}>
+      <Inner>
+        <Monitor>
+          <Background style={backgroundStyles}>{children}</Background>
+        </Monitor>
+        <Stand />
+      </Inner>
+    </Wrapper>
+  );
+});
+
+Desktop.defaultProps = {
+  backgroundStyles: null
+};
+
+Desktop.propTypes = {
+  backgroundStyles: propTypes.object,
+  // eslint-disable-next-line react/require-default-props
+  children: propTypes.node
```

Comment:
Isn't `children` a required prop on this case? If so we can just make it required and remove the `// eslint-disable-next-line react/require-default-props` comment. What do you think?

Reply:
children is not required, because most of the time the user will not pass any children. User will style Desktop background by providing `backgroundStyles` prop like: ```<Desktop backgroundStyles={{ background: 'url(image.png)}} />```. But we also want to give user a possibility to pass some other elements to the background, hence the children prop. If it's not required then there's no reason to put it in default props. I think I found similar approach in Material UI codebase. If it causes any problems in the future we can change it